x-pack/plugins/security_solution/public/management/pages/trusted_apps/view/components/condition_entry_input/index.tsx

```diff
@@ -66,6 +67,14 @@ export interface ConditionEntryInputProps {
   'data-test-subj'?: string;
 }
 
+// adding a style prop on EuiFlexGroup works only partially
+// and for some odd reason garbles up gridTemplateAreas entry
+const StyledEuiFlexGroup = styled(EuiFlexGroup)`
+  display: grid;
```

Comment:
I'd rather add these styles with a style prop below [condition_entry_input/index.tsx#L165](https://github.com/elastic/kibana/blob/f657b8058d901155295330ceb75c2292efa4722f/x-pack/plugins/security_solution/public/management/pages/trusted_apps/view/components/condition_entry_input/index.tsx#L165) as

```
<EuiFlexGroup style={{
display: 'grid',
gridTemplateColumns: '25% 25% 45% 5%',
gridTemplateAreas: 'field operator value remove'
}}>
```
but for some odd reason, the compiled code is devoid of `grid-template-areas` as if it doesn't care about `gridTemplateAreas` css property.

Reply:
@paul-tavares any ideas on the above?

Reply:
The Eui team says they don't support this fully. https://elastic.slack.com/archives/C7QC1JV6F/p1619080432325400. And they recommend that we use a custom `div` element for creating styles and not overriding component styles. I'll do that in a new commit if everyone agrees.

Reply:
Yeah, I am not sure that you want to alter the EuiFlexGroup to switch it from `display: flex` to `display: grid`.  Could you just add the percentages to the indivicual containers inside of the flex group as `width: `? woudl that work?

_(ps.: I forgot how cool and useful the CSS `grid` display is)_

Reply:
@paul-tavares just adding width to flex groups doesn't work, unfortunately. The css `grid` works across browsers though so I'm going to stick to it for now although will add it to a new `div` like Eui team suggests.

Reply:
So adding a `div` around the `EuiFlexItem`s, and adding grid styles to this new `div` doesn't just work with that simple change. Needs more CSS fixing. I think for now I like that the `EuiFlexGroup` is overridden plainly cause it's less code to maintain. We could either ask the Eui team to give us input fields that align nicely in such a layout or give us a component with a `grid` display that's currently not present in the Eui library.